src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/IndexQueueService.kt

```diff
@@ -1,50 +1,48 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.services
 
-import com.amazonaws.services.sqs.AmazonSQS
 import com.amazonaws.services.sqs.AmazonSQSAsync
 import com.amazonaws.services.sqs.model.PurgeQueueRequest
 import com.amazonaws.services.sqs.model.SendMessageRequest
 import com.google.gson.Gson
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.beans.factory.annotation.Qualifier
-import org.springframework.beans.factory.annotation.Value
 import org.springframework.data.domain.PageRequest
 import org.springframework.stereotype.Service
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.MissingQueueException
 
 @Service
 class IndexQueueService(
-  @Autowired @Qualifier("awsSqsIndexASyncClient") private val awsSqsIndexASyncClient: AmazonSQSAsync,
-  @Autowired @Qualifier("indexQueueUrl") private val indexQueueUrl: String,
-  @Qualifier("awsSqsIndexClient") private val indexAwsSqsClient: AmazonSQS,
-  @Qualifier("awsSqsIndexDlqASyncClient") private val awsSqsIndexDlqASyncClient: AmazonSQSAsync,
-  @Qualifier("awsSqsIndexDlqClient") private val indexAwsSqsDlqClient: AmazonSQS,
-  @Value("\${sqs.index.dlq.name}") private val indexDlqName: String,
+  hmppsQueueService: HmppsQueueService,
   private val gson: Gson
 ) {
 
-  val indexDlqUrl: String by lazy { indexAwsSqsDlqClient.getQueueUrl(indexDlqName).queueUrl }
+  private val indexQueue = hmppsQueueService.findByQueueId("indexqueue") ?: throw MissingQueueException("HmppsQueue indexqueue not found")
+
+  private val indexQueueSqsClient = indexQueue.sqsClient as AmazonSQSAsync
+  private val indexQueueSqsDlqClient = indexQueue.sqsDlqClient as AmazonSQSAsync
+  private val indexQueueUrl: String = indexQueue.queueUrl
+  private val indexDlqUrl: String = indexQueue.dlqUrl
 
   fun sendIndexRequestMessage(payload: PrisonerIndexRequest) {
-    awsSqsIndexASyncClient.sendMessageAsync(SendMessageRequest(indexQueueUrl, gson.toJson(payload)))
+    indexQueueSqsClient.sendMessageAsync(SendMessageRequest(indexQueueUrl, gson.toJson(payload)))
```

Comment:
Much of this PR involves standardizing the SQS client bean names throughout the project.